src/main/webapp/main.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
[Whole file] some of the indentation in this file is off.

Reply:
Yeah, there was a problem when I moved the files from codepen to cloudshell, but I fix it now.